### DISCUSSION TITLE
metadata-logs folder large

### DISCUSSION THREAD

**Discussion by ylnest2018** [advplyr/audiobookshelf#3602] [General]
Wondering if this is by design? if this can be cleared out? I have 12gb+ in logs alone.

**Comment by nichwall** [comment]
You can safely delete old log files. There is also https://github.com/advplyr/audiobookshelf/issues/3363

**Comment by advplyr** [comment]
Are these scan logs or daily logs? All the scan logs stick around so maybe we need to setup auto-cleanup for those.

**Reply by advplyr** [reply]
Daily logs files already get cleaned up. Only 7 days of daily logs are kept

**Reply by ylnest2018** [reply]
Daily logs are clearing however the metadata/logs/scans is 7months old. These are ok to delete?

**Reply by pwinnski** [reply]
All logs other than today's daily log can be deleted at any time without issue if you don't need them.
